src/app/children/health-checkup/health-checkup.component.html

```diff
@@ -0,0 +1 @@
+<app-entity-subrecord [records]="records" [columns]="columns" [newRecordFactory]="generateNewRecordFactory().bind(this)"></app-entity-subrecord>
```

Comment:
what does `.bind(this)` do?

Reply:
It was heritage code from some thing i tried. I fixed it.